samples/Localization.StarterWeb/Startup.cs

```diff
@@ -1,19 +1,20 @@
 using System;
```

Comment:
While we're messing around with the usings we may as well remove the couple unused ones in here.